pkg/sql/table.go

```diff
@@ -813,13 +814,35 @@ func (tc *TableCollection) validatePrimaryKeys() error {
 	return nil
 }
 
+// MigrationSchemaChangeRequiredContext flags a schema change as necessary to
+// run even in a mixed-version 19.2/20.1 state where schema changes are normally
+// banned, because the schema change is being run in a startup migration. It's
```

Comment:
Might be worth also noting this relies on/only works for places where the same or child `context.Context` is used in execution of the query i.e. the internal executor as used in migrations (and that may differ from the semanttics that some are used to when we've done similar flags via session vars)

Reply:
Yeah, this is a good point. I figure we can live without a comment saying so—we almost certainly won't have more migrations involving schema changes introduced in 20.1 (and if there are, we may have bigger problems), and it'll be hard to make a change that gets this wrong because the mixed-version logic test will panic immediately when the migration fails.